src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/SingleFileApiTests.cs

```diff
@@ -62,27 +69,10 @@ public void SelfContained_NetCoreApp3_CompatMode_SingleFile_APITests()
                 .And.NotHaveStdOutContaining("Microsoft.NETCore.App.deps.json") // No framework - it's self-contained
                 // For single-file, Environment.GetCommandLineArgs[0] should return the file path of the host.
                 .And.HaveStdOutContaining("Command line args: " + singleFile)
-                .And.HaveStdOutContaining("ExecutingAssembly.Location: " + extractionBaseDir.FullName) // Should point to the app's dll
-                .And.HaveStdOutContaining("AppContext.BaseDirectory: " + extractionBaseDir.FullName); // Should point to the extraction directory
-        }
-
-        [Fact]
-        public void GetCommandLineArgs_0_Non_Bundled_App()
-        {
-            var fixture = sharedTestState.TestFixture.Copy();
-            var dotnet = fixture.BuiltDotnet;
-            var appPath = BundleHelper.GetAppPath(fixture);
-
-            // For non single-file apps, Environment.GetCommandLineArgs[0]
-            // should return the file path of the managed entrypoint.
-            dotnet.Exec(appPath, "cmdlineargs")
-                .CaptureStdErr()
-                .CaptureStdOut()
-                .Execute()
-                .Should()
-                .Pass()
-                .And
-                .HaveStdOutContaining(appPath);
+                .And.HaveStdOutContaining("ExecutingAssembly.Location: " + extractionDir) // Should point to the app's dll
+                .And.HaveStdOutContaining("AppContext.BaseDirectory: " + extractionDir) // Should point to the extraction directory
+                // In extraction mode, we should have both dirs
+                .And.HaveStdOutMatching($"NATIVE_DLL_SEARCH_DIRECTORIES: .*{extractionDir}.*{bundleDir}");
```

Comment:
Huh, I assume the `Matching` is turning this into a regex, meaning that the `\`s now need to be escaped on Windows?